storage/bucket.go

```diff
@@ -782,6 +786,15 @@ type BucketWebsite struct {
 	NotFoundPage string
 }
 
+// BucketCustomPlacementConfig holds the bucket's custom placement
+// configuration for Custom Dual Regions. See
+// https://cloud.google.com/storage/docs/locations for more information.
```

Comment:
Is there a subsection we can link to in the future that's specifically about CDR? @danielbankhead

Reply:
Yep, here's the subsection: https://cloud.google.com/storage/docs/locations#location-dr

Reply:
Cool -- @cojenco if you can update to use that link that would be great.

Reply:
awesome, thanks folks!